utils/file_utils.py

```diff
@@ -54,20 +55,46 @@ def copy_target_into(target, into):
             shutil.copytree(target, into)   # copy directory
         else:
             shutil.copyfile(target, into)   # copy single file
+        if verbose:
+            print(' - Copy completed: from %s into %s' % (target, into))
     except (AssertionError, RuntimeError, EnvironmentError) as err:
         raise RuntimeError('Couldn\'t copy "%s" into "%s": %s' % (
             target, into, str(err)))
 
 
+def move_target(target, into, verbose=False):
+    """
+        Move target folder into new folder. NOTE: target will be removed!
+    """
+    if verbose:
+        print(' -- Moving %s into %s' % (target, into))
+    if target == into:
+        return False
+    copy_target_into(target, into, verbose=verbose)
+    orig_content = glob.glob(target + '/*')
+    copied_content = glob.glob(into + '/*')
+
+    # verified number of files of copied files with original
+    if len(orig_content) != len(copied_content):
+        if verbose:
+            print(' - Failed to move() %s into %s! Copied content is not the same.')
+
+        remove_target(into, verbose=verbose)
+        return False
+
+    # copied and original content seems to be the same. Thus, can remove target.
+    remove_target(target, verbose=verbose)
+    return True
+
+
 def make_dir(path):
     """
-        FIXME: all we need is ', exists_ok=True' and we can get rid of this.
         A simple wrapper around os.makedirs that skip existing folders.
     :param 'path': [str] Leave this routine with a (new) directory at path
     :return: None or raised error
     """
     try:
-        os.makedirs(path)
+        os.makedirs(path, exist_ok=True)
```

Comment:
Move "exist_ok=True" up into the make_dir() declaration, then this call becomes

os.makedirs(path, exist_ok=exist_ok)